auth/AuthenticatorECDSA.m

```diff
@@ -1,20 +1,42 @@
 classdef AuthenticatorECDSA < Authenticator
     %AUTHENTICATORECDSA Summary of this class goes here
     %   Detailed explanation goes here
-    
+    properties(Access = private)
+     private_key
```

Comment:
remove private_key and public_key. In the super class Authenticator properties, change them to (Access=protected) so that you can use them. Implement the methods to interpret the public/private keys in the *super* class so we can recycle these methods if Todd asks us to implement ECschnorr.

Reply:
unit test:
construct authenticator with a random private key,
pass public key to new authenticator, test sign and verify